prereise/gather/griddata/hifld/data_process/topology.py

```diff
@@ -73,25 +116,53 @@ def connect_islands_with_minimum_cost(
     :param pandas.DataFrame subs: substation data frame indexed by ``ID`` with two
         columns ``LATITUDE`` and ``LONGITUDE`` representing the geographical
         coordination of each entry.
-    :param function cost_metric: a function defines the cost metric to calculate the
-        weight of lines, defaults to :py:func:`haversine`.
+    :param int island_size_lower_bound: smallest island the function should consider
+        exclusively, defaults to 0.
+    :param int island_size_upper_bound: largest island the function should consider
+        exclusively, defaults to None.
```

Comment:
Does a default of `None` mean that all islands are considered?

Reply:
Yes, as long as i`sland_size_lower_bound == 0`.